package.json

```diff
@@ -55,12 +64,17 @@
     },
     "homepage": "https://thumbprint.design/tokens",
     "dependencies": {
+        "@netlify/functions": "^1.0.0",
         "apollo-server-lambda": "^2.9.7",
         "graphql": "^14.5.8"
     },
+    "resolutions": {
+        "@types/istanbul-lib-report": "3.0.0"
```

Comment:
Why is this needed?

Reply:
I had the same issue in the icons conversion. It's a transitive dependency of TS Jest, and it causes the script to fail with a type error if you use the default version it ships with.